packages/corvid-types/index.js

```diff
@@ -0,0 +1,19 @@
+const prettyStringify = content => JSON.stringify(content, null, 2);
+const getTsConfigContent = configName =>
+  prettyStringify({
+    extends: `corvid-types/configs/tsconfig.${configName}.json`
+  });
+
+const getPageDynamicTypingsContent = elementsMap =>
+  prettyStringify(
```

Comment:
maybe add a comment that will tell users not to write the types.d.ts and tsconfig.json file because we overwrite those files each save